mesh/agents/twitter_info_agent.py

```diff
@@ -240,6 +314,53 @@ def _type(t: Dict) -> str:
 
         return result
 
+    async def _practicaltools_call(self, endpoint: str, parameters: Dict[str, Any]) -> Dict[str, Any]:
+        if not self.apify_client:
```

Comment:
such checks should have been completed in the constructor. don't check again in functions - same principle applies to other functions